torch/distributed/utils.py

```diff
@@ -31,7 +34,7 @@ def to_map(obj):
                     current_stream.wait_stream(stream)
                     # Ensure tensor memory is not reused until work on
                     # main stream is complete
-                    output.record_stream(current_stream)  # type: ignore[arg-type]
+                    output.record_stream(current_stream)  # type: ignore[attr-defined]
```

Comment:
Was the mypy ignore comment meant to be changed? (asking since now mypy is complaining)